### PR TITLE
Add GITEA_CUSTOM configuration in customization section

### PR DESCRIPTION
**Pull request by blankoworld**
Fix https://github.com/go-gitea/gitea/issues/1452 about missing GITEA_CUSTOM explanation in documentation

### CODE REVIEW

File: content/doc/advanced/customizing-gitea.en-us.md [hunk 1]
```diff
@@ -17,6 +17,14 @@ menu:
 
 The main way to customize Gitea is by using the `custom` folder. This is the central place to override and configure features.
 
+Use `GITEA_CUSTOM` to define another folder instead. For example:
+
+```
+GITEA_CUSTOM=/srv/gitea/mycustom ./gitea web
+```
+
+**Note** that by default Gitea will use the `custom` folder in the current working directory.
```
Comment (strk) [inline content/doc/advanced/customizing-gitea.en-us.md]: I'm afraid at the moment it'll be the `custom` folder in the directory containing the `gitea` binary unless `GITEA_WORK_DIR` env is set, in which case that's the directory used to resolve relative paths like `custom/`

**Review by strk — CHANGES_REQUESTED** [review]
Please check the resolution of `custom/` dir

File: content/doc/advanced/customizing-gitea.en-us.md (hunk 1 above)
Comment (blankoworld) [reply content/doc/advanced/customizing-gitea.en-us.md]: You're right, I didn't be aware that GITEA_WORK_DIR variable exists.
That's why it should be better to create a specific subsection in this page to explain how GITEA_WORK_DIR and GITEA_CUSTOM works.
Or, in another case: create a specific page to list all available environment variable for GITEA?
What's your opinion about that?

**Comment by strk** [comment]
I think a section about environment variables will be great,
then you could reference those env variables at relevant places.

File: content/doc/advanced/customizing-gitea.en-us.md (hunk 1 above)
Comment (strk) [reply content/doc/advanced/customizing-gitea.en-us.md]: Yes please add a section about env variables

**Comment by lunny** [comment]
LGTM

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
LGTM

**Pull request merged**